### PR TITLE
Add Ball Classification Feature

### PR DESCRIPTION
**Pull request by brandonabela**

### CODE REVIEW

File: Logic/item_classification.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    def get_white_count(ball_pixels):
+        '''Finding the number of white pixels within the ball pixels'''
+
+        white_count = 0
+
+        for pixel in ball_pixels:
+            is_r_valid = 192 <= pixel[0] <= 255
+            is_g_valid = 192 <= pixel[1] <= 255
+            is_b_valid = 192 <= pixel[2] <= 255
+
```
Comment (KeithGeorgeCiantar) [inline Logic/item_classification.py:46]: The values [0], [1] and [2] correspond to BGR not RGB. Just swap is_r_valid with is_b_valid. But essentially no difference.

File: Logic/item_classification.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    def get_black_count(ball_pixels):
+        '''Finding the number of black pixels within the ball pixels'''
+
+        black_count = 0
+
+        for pixel in ball_pixels:
+            is_r_valid = 0 <= pixel[0] <= 64
+            is_g_valid = 0 <= pixel[1] <= 64
+            is_b_valid = 0 <= pixel[2] <= 64
+
```
Comment (KeithGeorgeCiantar) [inline Logic/item_classification.py:62]: Same as previous

File: Logic/item_classification.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        return black_count
+
+    @staticmethod
+    def is_solid_ball(white_count, black_count):
+        '''Checking whether the pixel count is a solid ball'''
+
+        return 7 <= white_count <= 72 and 0 <= black_count <= 41
+
+    @staticmethod
+    def is_stripped_ball(white_count, black_count):
```
Comment (KeithGeorgeCiantar) [inline Logic/item_classification.py]: The function name should be is_striped_ball. There is a small typo.

**Review by KeithGeorgeCiantar — COMMENTED** [review]
Very good work, just some small typos.

File: Logic/item_classification.py (hunk 1 above)
Comment (brandonabela) [reply Logic/item_classification.py:46]: Done

File: Logic/item_classification.py (hunk 2 above)
Comment (brandonabela) [reply Logic/item_classification.py:62]: Done

File: Logic/item_classification.py (hunk 3 above)
Comment (brandonabela) [reply Logic/item_classification.py]: Done

**Review by KeithGeorgeCiantar — APPROVED** [review]

**Pull request merged**